hack/local-up-cluster.sh

```diff
@@ -38,7 +38,7 @@ KUBECTL=${KUBECTL:-cluster/kubectl.sh}
 WAIT_FOR_URL_API_SERVER=${WAIT_FOR_URL_API_SERVER:-10}
 ENABLE_DAEMON=${ENABLE_DAEMON:-false}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"127.0.0.1"}
-
+KUBELET_SYNC_FREQUENCY=${KUBELET_SYNC_FREQUENCY:-"1m0s"}
```

Comment:
yup, lets add comment 
`KUBELET_SYNC_FREQUENCY=${KUBELET_SYNC_FREQUENCY:-"1m0s"} # 1 minute so that sync happens faster` or whatever (not sure why you picked 1m in particular?)`